src/edu/wright/cs/raiderplanner/controller/MenuController.java

```diff
@@ -148,7 +148,7 @@
 	private DropShadow moduleHoverShadow = new DropShadow(screenAverage * 0.02, 0, 0, Color.BLACK);
 	private InnerShadow modulePressedShadow = new InnerShadow(screenAverage * 0.017, 0, 0,
 			Color.BLACK);
-
+	private Stage  stage = null;
```

Comment:
Please remove the extra space between `Stage` and `stage`.

Reply:
How did you even see that extra white space!